sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -834,7 +1063,81 @@ std::vector<node> get_root_nodes() const;
 
 |===
 
-Table {counter: tableNumber}. Member functions of the `command_graph` class for queue recording.
+Table {counter: tableNumber}. Member functions of the `command_graph` class for
+graph update.
+[cols="2a,a"]
+|===
+|Member function|Description
+
+|
+[source,c++]
+----
+void update(node& node);
+----
+
+| Updates an executable graph node that corresponds to `node`. `node` must be a
+kernel execution node. Kernel arguments and the ND-range of the node will be
+updated inside the executable graph to reflect the current values in `node`.
+
+Updating these values will not change the structure of the graph.
+
+Note: The implementation may perform a blocking wait during this call on any
```

Comment:
I'd remove the "Note:" prefix, I don't think it adds anything other than ambiguity over whether this is normative text or not.